Library/Formula/lua.rb

```diff
@@ -13,7 +13,8 @@ class Lua < Formula
     sha256 "519abb38296981baf49dd00ac80a2ac742f8e278b0b616d88d7fab8702ad430d" => :mountain_lion
   end
 
-  def pour_bottle?
+  # Doesn't need a reason; just behaves like it doesn't have `:cellar any`.
+  pour_bottle? nil do
```

Comment:
The end of the comment should probably read “[…] \`cellar :any\`.” instead. Would be good to provide a reason. What's wrong with stating that it needs the prefix to be `/usr/local`?